app/src/main/java/org/fossasia/openevent/general/di/Modules.kt

```diff
@@ -193,6 +199,16 @@ val networkModule = module {
             .baseUrl(baseUrl)
             .build()
     }
+    single<Retrofit>("autoPlace_api") {
```

Comment:
Why not using geocoding jar from mapbox?

Reply:
Do you mean extracting jar from MapBox Sdk(that added for Autocomplete) ?

Reply:
Since last time adding Sdk introduced too much size
I thought too use Api request method as we only need the list of autocomplete places . We would have the full control over code 
Plus we wont have no extra methods and size of MapBox Java Sdk

Reply:
These are api provided with Mapbox java sdk services https://docs.mapbox.com/android/api/mapbox-java/libjava-services/3.0.1/index.html 

I think I can't select or exclude other packages in services dependency.

Reply:
Use pure java lib of MapBox geocoding. We don't want to maintain API classes of MapBox. Implement your own solution only if the jar contains mapbox-gl-native, which I suppose it won't

Reply:
I understand. I checked doesn't include mapbox-gl-native
proceeding with Mapbox java sdk then